pom.xml

```diff
@@ -915,5 +925,37 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>integration-test</id>
+            <!-- This profile can be used to run integration tests all by themselves without running the unit tests. -->
+            <!-- Use the command "mvn verify -Pintegration-test -Dquickly=true" to run the integration tests. -->
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-failsafe-plugin</artifactId>
+                        <version>${maven-failsafe-plugin-version}</version>
+                        <executions>
+                            <execution>
+                                <goals>
+                                    <goal>integration-test</goal>
+                                    <goal>verify</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                        <configuration>
+                            <skipITs>false</skipITs>
+                            <includes>
+                                <include>**/*IT.java</include>
+                                <include>**/*IntegrationTest.java</include>
+                            </includes>
+                            <excludes>
+                                <exclude>**/*Test.java</exclude>
+                            </excludes>
+                        </configuration>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
```

Comment:
I am truly sorry for having missed this, but I would us to avoid piling up the build with profiles that are very rarely used (by ourselves or by our CI or by the automation that we have). 

We (myself included) spent a lot of time cleaning up a several profiles that existed on Camel 3 but we never used and complicated the build.

For profiles, the less and simpler, the better.  

@davsclaus @oscerd wdyt?

Reply:
I think it's better to avoid more profiles, we have already enough

Reply:
Has anybody else tried to run integration tests lately? When I run `mvn failsafe:verify` against `main`, I get dozens of these errors:

```
[WARNING] The POM for org.apache.maven.plugins:maves-surefire-plugin:jar:3.2.5 is missing, no dependency information available
[WARNING] Failed to retrieve plugin descriptor for org.apache.maven.plugins:maves-surefire-plugin:3.2.5: Plugin org.apache.maven.plugins:maves-surefire-plugin:3.2.5 or one of its dependencies could not be resolved: The following artifacts could not be resolved: org.apache.maven.plugins:maves-surefire-plugin:jar:3.2.5 (absent): org.apache.maven.plugins:maves-surefire-plugin:jar:3.2.5 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced
```

It turns out that this line in `pom.xml`

                    `<artifactId>maves-surefire-plugin</artifactId>`

should be this:

                    `<artifactId>maven-surefire-plugin</artifactId>`

When I fix that, no integration tests are executed. Maybe there is a magic combination of properties and profiles that enables integration tests to run, but I couldn't figure it out.

You make a very good point about reducing complexity, but I think maybe there needs to be a separate effort to refactor the POM structure. Things seem to have gotten so complicated that tests can't even run reliably.  In the meantime, after days of trying, the profile that I created is the only way I could find to run integration tests.

Reply:
You should use: `mvn verify` to run the tests (unit + integration).

Reply:
My (and I guess, the same for other committers) is just: 

```
cd path/to/project 
# Build with Maven Daemon, for quicker build
mvnd -Dquickly clean install
# Go to whatever you are working on (i.e.; camel-kafka)
cd components/camel-kafka 
# Run the tests
mvn verify 
```

Done.